requirements.txt

```diff
@@ -0,0 +1 @@
+coala_bears>=0.10.0
```

Comment:
@RohanVB review history showed that you went for some dev version of coala-bears, so I assume there might've been some reason for opting for that one. If it is, consider it a gentle reminder, if it is not, ignore ;)